angular-template.js

```diff
@@ -7,5 +7,28 @@ module.exports = {
             Overrides to Angular template recommended rules:
             https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin-template/src/configs/recommended.json
         */
+
```

Comment:
We discussed the accessibility rules in our meeting today. Our decision was to disable all of them for now in our recommended config. This (sadly) most accurately reflects where the company is today with accessibility--it's not something that developers should spend time enforcing.

There is an upcoming push from the Design team and the design system to start to care more. At that time we can revisit this decision and hopefully turn the rules on with the backing of leadership to spend developer time making apps accessible.

@TrevorKarjanis can you update the PR to reflect this with a comment something like this?
"At the moment, there's no NI-wide requirement to enforce accessible apps. These rules can be useful if individual apps choose to prioritize it. They may be enabled in the future as the priority of accessibility increases."

Reply:
Yes, and I would also recognize that this is already reflected in our JavaScript and TypeScript configurations.